frontend-cert/js-projects/lab-madlibs/user-stories.md

```diff
@@ -0,0 +1,15 @@
+1. You should define the following variables using `let`:
+- `adjective`
+- `noun`
+- `verb`
+- `place`
+- `adjective2`
+- `noun2`
+
+1. You should assign the above variables some string values of your choice.
+
+1. You should use the following story template to create a story:
```

Comment:
Will the variable `let story` be given or will the learner be required to define it?

Reply:
The users should use the provided story template and assign it to any variable of their choice. It could be `story` or anything else.

Reply:
@zairahira 

it will probably be easier to test to have them declare a `story` variable.

Reply:
added!